.gitignore

```diff
@@ -0,0 +1,4 @@
+.vscode/settings.json
```

Comment:
Can we just ignore the whole of .vscode ? (I think IDE stuff in repos is a no-no)